launchers/launch.sh

```diff
@@ -20,4 +20,4 @@ done
 
 echo "Killing OmniSharp server"
 
-kill_and_be_killed $$
```

Comment:
My OCD prone sublime automatically removes superfluous whitespace from line endings. That's the only change to this line.